### PR TITLE
[1LP][RFR] Replacing raw ovirtsdk api calls by wrapanapi api

### PR DESCRIPTION
**Pull request by quarckster**
Intent
==================
This PR attempts to remove all raw ovirtsdk api calls by wrapanapi calls. It depends on  ~https://github.com/ManageIQ/wrapanapi/pull/234~ ~https://github.com/ManageIQ/wrapanapi/pull/263~, . Requires version wrapanapi > 2.8.12

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 82fb6572f05c8d13789a9b4e93a0beba1f4038b1:

`scripts/template_upload_rhevm_qcow2.py`:
- :red_circle: Line 18:1: **F401** *'ovirtsdk4.types' imported but unused*
- :bangbang: Line 473:101: **E501** *line too long (101 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eebc151a524272760c8027c07640e4414708ccf2:

`scripts/template_upload_rhevm_qcow2.py`:
- :bangbang: Line 468:101: **E501** *line too long (101 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/utils/virtual_machines.py [hunk 1]
```diff
@@ -6,15 +6,13 @@
 from cfme.utils.providers import get_crud
 from cfme.fixtures.pytest_store import store
 from novaclient.exceptions import OverLimit as OSOverLimit
+from wrapanapi.rhevm import Error as RHEVRequestError
 from ssl import SSLError
 from cfme.utils.log import logger
 from cfme.utils.mgmt_system import exceptions
```
Comment (mshriver) [inline cfme/utils/virtual_machines.py]: This import is breaking travis it looks like.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit defc3f07390c20f8c572c2148d50db472c1e2a2c:

`scripts/template_upload_rhevm.py`:
- :bangbang: Line 276:101: **E501** *line too long (110 > 100 characters)*
- :red_circle: Line 410:58: **F821** *undefined name 'temp_template'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 87f70b235107129382c335aeade2f0ffd7d894df:

`docs/conf.py`:
- :bangbang: Line 281:91: **E231** *missing whitespace after ','*
- :bangbang: Line 281:92: **E999** *SyntaxError: invalid syntax*

`scripts/template_upload_rhevm.py`:
- :bangbang: Line 276:101: **E501** *line too long (110 > 100 characters)*
- :red_circle: Line 410:58: **F821** *undefined name 'temp_template'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 354bfaf1a4cb2a3abfe268e36e757d8ddab0c644:

`scripts/template_upload_rhevm.py`:
- :bangbang: Line 276:101: **E501** *line too long (110 > 100 characters)*
- :red_circle: Line 410:58: **F821** *undefined name 'temp_template'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**